docs/versioned_docs/version-0.12.2/general/02-Getting Started/getting-started-linux.md

```diff
@@ -12,22 +12,43 @@ pagination_next: general/Getting Started/write-your-first-instruction
 - A text editor/IDE like [Visual Studio Code](https://code.visualstudio.com/)
 - :warning: [Information for Wayland](../07-Troubleshooting/linux.md#wayland).
 
-## Access Token
+## Installation
 
-As we need to prevent misuse of our API, we need you to create some credentials through our __AskUI Studio__ (for free).
+Open a directory where you would like to place the AskUI setup. If you haven’t already, set up an NPM project by running:
 
-Please [fill out this form](https://xa5a040gvfz.typeform.com/to/Ndh2NkV6) to schedule a demonstration or [request a trial](https://xa5a040gvfz.typeform.com/to/IHdr0qY5) to obtain access to __AskUI Studio__.
+```shell
+npm init -y
+```
 
-## Installation
+This will create a `package.json` file inside your present working directory which contains the configuration of your project including a description of its dependencies.
 
-Open a directory where you would like to place the AskUI setup and run this command. Answer all the questions:
+Install `askui` and other dependencies for writing and executing tests:
 
 ```shell
-npx askui@latest init -s linux
+npm i -D askui @askui/askui-reporters typescript ts-node @types/jest ts-jest jest
 ```
 
+<details>
+  <summary>Quick explanation of all the dependencies</summary>
+
+- [AskUI](https://www.npmjs.com/package/askui): Controlling a multitude of operating systems with commands based on automatically detected screen elements etc.
+- [@askui/askui-reporters](https://www.npmjs.com/package/askui-reporters): AskUI reporters for nice looking reports.
+- [typescript](https://www.npmjs.com/package/typescript): Allowing you to write your tests in [TypeScript](https://www.typescriptlang.org/) instead of JavaScript
+- [ts-node](https://www.npmjs.com/package/ts-node): TypeScript execution and REPL for node.js, with source map and native ESM support.
+- [jest](https://www.npmjs.com/package/jest): Allowing you to write and run tests (`describe`, `it`, assertions, mocking etc.)
+- [ts-jest](https://www.npmjs.com/package/ts-jest): A Jest transformer with source map support that lets you use Jest to test projects written in TypeScript
+- [@types/jest](https://www.npmjs.com/package/@types/jest): Types for Jest
+
+</details>
+
 :::note
 
 To create and serve a static HTML-Report you have to install [Allure](https://github.com/allure-framework/allure2#download) and then call `allure serve ./allure-results` from your root-directory.
 
 :::
+
+## Access Token
+
+As we need to prevent misuse of our API, we need you to create some credentials through our __AskUI Studio__ (while still free) and [configure our library to use these credentials](../../api/Configuration/askui-ui-control-client#credentials) for authenticating and authorizing with our API.
```

Comment:
Well-explained :) Best policy is transparency :)

Reply:
Actually I think this paragraph was your creation over a year ago 😄